_games/cave_story.md

```diff
@@ -0,0 +1,9 @@
+---
+short-name: cave_story
+long-name: Cave Story
+randomize:  |
+            - All items can be randomized
+need-to-play:   |
+                - Windows or Wine. The game is included in Randovania
```

Comment:
Tweaked doesnt require WINE. do we wanna change this now or later?

Reply:
later, I'd rather it be a separate PR